### PR TITLE
ci: add missing safedir setup for commit and copyright check

### PR DESCRIPTION
**Pull request by MrAnno**
Sorry, I misread these two.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

File: .github/workflows/publish-docker-image.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
         id: tags
         run: |
           if [[ '${{ inputs.pkg-type }}' = 'stable' ]]; then
-            TAGS='${{ steps.stable-tags.outputs.tag }}'
+            TAGS='${{ steps.stable-tags.outputs.tags }}'
```
Comment (bazsi) [inline .github/workflows/publish-docker-image.yml:48]: Ah, I think we will definitely need some kind of mechanism for testing the "stable" pipeline before doing the actual release. I don't see how else we would be catching these kind of errors.

**Review by bazsi — APPROVED** [review]

**Comment by MrAnno** [comment]
This fixes false-negative CI commit style check results. @HofiOne @bazsi @therandomstring Feel free to merge it.

File: .github/workflows/publish-docker-image.yml (hunk 1 above)
Comment (MrAnno) [reply .github/workflows/publish-docker-image.yml:48]: Yeah, we previously tested these jobs manually after each change.
We could do some release CI tests using a test APT repo/a test docker namespace, etc.

**Pull request merged**